src/parser.js

```diff
@@ -22,13 +22,24 @@ define([
     }
 
     function getAttributes (element) {
-        var attributes = $(element)[0].attributes,
-            attrMap = {};
+        var attributes = _.chain($(element)[0].attributes)
+                .map(function (value) {
+                    return [value.nodeName, value.nodeValue];
+                }) .object().value(),
+            attrKeys = _.keys(attributes);
+
+        _.each(attrKeys, function (key) {
+            if (key.startsWith('vellum:') &&
```

Comment:
Looks like this causes all `vellum:attrname` attribute values (except the excluded ones) to be moved to `attrname`. Are we sure we want that for every `vellum:` attribute (except for the excluded ones) going forward? Can we white-list attributes that we want to be translated in this way rather than automatically translating every single attribute (except for `vellum:ignore` and `vellum:lock`, of course) that starts with `vellum:`?